nautobot/core/ui/object_detail.py

```diff
@@ -729,6 +729,66 @@ def render_body_content(self, context):
         return result
 
 
+class StatsPanel(Panel):
+    def __init__(
+        self,
+        *,
+        stats=None,
+        filter_name=None,
```

Comment:
From the template above, it seems like `filter_name` should be a required argument?